### PR TITLE
Project: Restaurant Page

### PR DESCRIPTION
**Pull request by macnick**
In this project I created a restaurant page. The objective is to dynamically render every page using Javascript DOM manipulation.
Here is what I did:

1. installed webpack
2. installed linters
3. Created 2 directories src and dist
4. Added index.js to scr
5. Added index.html and directories for css, images and fonts in dist directory
6. Created a modules directory inside the src and put all the modules for the page creation
7. index.js calls pageCreate() to create the page

Live page is not yet ready. I will fix it after the project is approved and will make it run on github

Thank you for reading all this. Please review my code.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (shloch) [inline .eslintrc.json]: Please use same config file as the one provided by microverse 
`https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json`
When you done, re-run for linting errors.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Javascript generated site",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (shloch) [inline package.json]: Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. [ [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) ]

Modify this section to something like this code below : 
``` 
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @macnick 

Very practical home page message lol
Screen Shot 2020-03-25 at 2 24 49 PM

Your code is clean.......nothing to say about your project, it's just cool.
### However, to get thumbs up.........please round up with some of the observations below :

File: .eslintrc.json (hunk 1 above)
Comment (macnick) [reply .eslintrc.json]: Done! re-run for errors. Found none.

File: package.json (hunk 2 above)
Comment (macnick) [reply package.json]: Oops! I missed that. Thanks for pointing out!

**Review by bellom — APPROVED** [review]
## Hi @macnick  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by macnick** [comment]
Thank you so much Bello

**Pull request merged**